### ISSUE TITLE
Is brave dead?

### ISSUE THREAD

**Issue by SonnyWalkman** [bbc/brave#88]
Has there been anymore discussion and work with replacing intervideo interaudio with ridgeruns interpipes?

**Comment by nikell28** [comment]
@SonnyWalkman 
Dead , more than completely

**Comment by RobertLowe** [comment]
How did it die? Did BBC axe it, or just no interest anymore, or etc?

**Comment by nikell28** [comment]
Unfortunately, I am not aware of the motives. There is no podderka and it is impossible to reach the maintainer

**Comment by matthew1000** [comment]
👋  Brave is a prototype ... it's not had much evolution because other internal systems were developed instead.

I'm hopeful that time can be found to update this prototype; but alas priorities have been elsewhere of recent.

**Comment by RobertLowe** [comment]
@matthew1000 can you provide any insight into how those systems changed? I'm curious if gstreamer is still used and if python is still used in that context? Or some new implementation appeared.

Just interested in tech around this, any big picture is much appreciated

**Comment by matthew1000** [comment]
@RobertLowe Yes, the internal BBC system is quite similar to Brave. It uses GStreamer and Python. It uses interpipes too which is more reliable as previously mentioned. The primary difference from Brave is that it's designed to fit in with other BBC systems - there's quite a bit of bespoke integration.

We've now done hundreds of hours of live streaming on the BBC website this way. GStreamer is awesome.

**Comment by SonnyWalkman** [comment]
@matthew1000, I agree, Gstreamer is awesome. It's a dream come true to live AV..

**Comment by hatsch** [comment]
> wave Brave is a prototype ... it's not had much evolution because other internal systems were developed instead.
> 
> I'm hopeful that time can be found to update this prototype; but alas priorities have been elsewhere of recent.

we are in the progress to build a gstreamer based (live) playout for our community TV station. It will be heavily based on BRAVE ( if not a fork ) we plan to incorporate suggestions above to use interpipes but i wonder if there's a realistic chance that you find time to share  some of that changes you've made to make BRAVE more reliable in the near future? @matthew1000 

Anyway gstreamer is awesome, so is BRAVE. Thank you
